Networking/NetworkingTests/Remote/StripeRemoteTests.swift

```diff
@@ -0,0 +1,309 @@
+import XCTest
+@testable import Networking
+
+
+/// Stripe Remote Unit Tests
+///
+final class StripeRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    let network = MockNetwork()
+
+    /// Dummy Site ID
+    ///
+    let sampleSiteID: Int64 = 1234
+
+    /// Repeat always!
+    ///
+    override func setUp() {
+        network.removeAllSimulatedResponses()
```

Comment:
nit: we call `super` in these overrides most of the time in case of any setup in the superclass
```suggestion
        super.setUp()
        network.removeAllSimulatedResponses()
```

Reply:
Fixed in both StripeRemoteTests and WCPayRemoteTests where the omission originated

https://github.com/woocommerce/woocommerce-ios/pull/5799/commits/ee329400901521dfe1a0e2cd4f000183272d2023